### ISSUE TITLE
Init: allow users to choose between local and cloud when not setting a cloud 

### ISSUE THREAD

**Issue by DanielMSchmidt** [hashicorp/terraform-cdk#694]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Currently init defaults to a cloud backend if the user has a cloud token. You can work around by using the `--local` flag, but it was quite surprising to me that I could not select this like I could when I hadn't logged in yet. I'd prefer a `--cloud` option and a selection like as if no token is set, so that the workflow stays the same.

**Comment by skorfmann** [comment]
> I'd prefer a `--cloud` option and a selection like as if no token is set, so that the workflow stays the same.

You mean some sort of dialog in the init workflow which asks for `local` / `cloud` ?

**Comment by DanielMSchmidt** [comment]
> You mean some sort of dialog in the init workflow which asks for local / cloud?

Exactly, there seems to be one that exists already when no terraform cloud credentials exist in the system
